src/all/pandachaika/src/eu/kanade/tachiyomi/extension/all/pandachaika/PandaChaika.kt

```diff
@@ -42,6 +44,9 @@ class PandaChaika(
 
     private val json: Json by injectLazy()
 
+    private val fakkuRegex = Regex("""(?:https?://)?(?:www\.)?fakku\.net/manga/""")
```

Comment:
Doesn't match the `baseLink` used further down. Did you mean to support both?
```suggestion
    private val fakkuRegex = Regex("""(?:https?://)?(?:www\.)?fakku\.net/(?:manga|hentai)/""")
```

Reply:
Thank you. But does `fakku.net/manga/` exist?

Reply:
Doesn't seem like it. Kinda just assumed it was a thing :stuck_out_tongue: 

I guess the correct regex would in that case be:
```suggestion
    private val fakkuRegex = Regex("""(?:https?://)?(?:www\.)?fakku\.net/hentai/""")
```